### PR TITLE
Adds dockerfile for Centos7.3 aarch64 builds

### PR DESCRIPTION
**Pull request by iahmad-khan**

### CODE REVIEW

File: slc7-builder/Dockerfile.arm64 [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+                   perl-Time-Local perl-Storable glibc-headers perl-threads \
+                   perl-Thread-Queue perl-Module-ScanDeps perl-Test-Harness \
+                   perl-Env perl-Switch perl-ExtUtils-Embed ncurses-libs \
+                   perl-libs nspr nss nss-util file file-libs readline \
+                   zlib popt bzip2 bzip2-libs python-devel java-1.8.0-openjdk \
+                   readline-devel perl-Switch perl-Storable perl-Env packages \
+                   perl-Thread-Queue  tk-devel tcl-devel which python2-pip \
+                   file git subversion bc finger zip unzip nano hostname \
+                   vim wget screen time && yum clean all
+RUN groupadd -g 1399 zh && adduser -u 50772 -g 1399 davidlt
```
Comment (davidlt) [inline slc7-builder/Dockerfile.arm64]: Remove my user from the image.

File: slc7-builder/Dockerfile.arm64 [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+RUN install -d -o root -g zh /build
```
Comment (davidlt) [inline slc7-builder/Dockerfile.arm64]: Do we need this?

File: slc7-builder/Dockerfile.arm64 [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+RUN adduser -u 22527 -g 1399 cmsbuild && install -d /opt && install -d -o cmsbuild /opt/cms /build
+RUN install -d -o cmsbuild /build
+RUN pip install --upgrade pip
+RUN mkdir -p /opt/py2-packages
+RUN pip install -t /opt/py2-packages pyyaml
+RUN pip install -t /opt/py2-packages PyGithub requests
+RUN chown -R cmsbuild /opt/py2-packages/*
+env PYTHONPATH /opt/py2-packages
+WORKDIR /build
+USER cmsbuild
```
Comment (davidlt) [inline slc7-builder/Dockerfile.arm64]: Where is the dumb-init?

File: slc7-builder/Dockerfile.arm64 [hunk 4]
```diff
@@ -0,0 +1,41 @@
+FROM centos/aarch64
+ADD cernca.repo /etc/yum.repos.d/cernca.repo
+ADD  GPG-KEY-EUGridPMA-RPM-3 /etc/pki/rpm-gpg/GPG-KEY-EUGridPMA-RPM-3
+RUN rpm -iUv http://dl.fedoraproject.org/pub/epel/7/aarch64/e/epel-release-7-9.noarch.rpm
+RUN rpm -iUvh http://linuxsoft.cern.ch/cern/slc65/updates/i386/RPMS/CERN-CA-certs-20160421-2.slc6.noarch.rpm
+RUN yum update -y && yum clean all
```
Comment (davidlt) [inline slc7-builder/Dockerfile.arm64]: Maybe we should be: yum clean all & yum makecache && yum update -y && yum clean all or similar here

Comment (davidlt) [inline slc7-builder/Dockerfile.arm64]: You can grab it from here: `http://davidlt.web.cern.ch/davidlt/vault/dumb-init/v1.2.0/dumb-init`

File: slc7-builder/Dockerfile.arm64 [hunk 5]
```diff
@@ -25,7 +25,7 @@ RUN yum install -y glibc coreutils bash tcsh zsh perl tcl tk readline \
                    readline-devel perl-Switch perl-Storable perl-Env packages \
                    perl-Thread-Queue  tk-devel tcl-devel which python2-pip \
                    file git subversion bc finger zip unzip nano hostname \
-                   vim wget screen time && yum clean all
+                   vim wget screen time voms-clients-cpp && yum clean all
```
Comment (davidlt) [inline slc7-builder/Dockerfile.arm64]: Don't you need `/etc/voms/{cms-voms2.cern.ch,cms-lcg-voms2.cern.ch}` if you plan to use `-voms cms` argument in `voms-proxy-init`.

Comment (davidlt) [inline slc7-builder/Dockerfile.arm64]: Ignore my previous comment, you can bind-mount within the container the certificates and vomses from here: `/cvmfs/grid.cern.ch/etc/grid-security`

Comment (smuzaffar) [inline slc7-builder/Dockerfile.arm64]: yes, binding -v /cvmfs/grid.cern.ch/etc/grid-security/vomses:/etc/vomses -v /cvmfs/grid.cern.ch/etc/grid-security:/etc/grid-securit worked.

Comment (davidlt) [inline slc7-builder/Dockerfile.arm64]: I think, we coulda also mount `/etc/grid-security/vomsdir`

Comment (smuzaffar) [inline slc7-builder/Dockerfile.arm64]: that is already mounted as it is under /cvmfs/grid.cern.ch/etc/grid-security

**Pull request merged**